test/removeNPMAbsolutePaths.js

```diff
@@ -375,6 +375,24 @@ describe('removeNPMAbsolutePaths.js', function () {
               expect(packageJson).to.have.property('_shasum');
             });
         });
+
+        it('rewrite file if contains _fields in package.json with newline', function () {
```

Comment:
Neither this test or any of the other tests actually checks whether the newline is preserved or not. Otherwise, you would have noticed the issue about `concat` that I mentioned in my other comment. 🙂

Reply:
@juanjoDiaz Indeed, looks like this case only happens on Windows platforms :D
Maybe we could do some matrix build on Travis?
Anyway the previous steps have been fixed.